postgres/functions/refresh_matviews.sql

```diff
@@ -4,11 +4,13 @@ CREATE OR REPLACE FUNCTION public.refresh_app_monitoring_matviews()
 AS $function$
 DECLARE
   matview RECORD;
+  start_time timestamp with time zone;
 BEGIN
+  start_time := transaction_timestamp();
```

Comment:
I think this will not change until the transaction ends which I doubt is the behaviour you want.

```
the transaction_timestamp() function returns the current date and time (including the time zone offset), at the start of the current transaction.
```

I think you would be best served by `clock_timestamp()` here as well.

Reply:
done